Base/usr/share/man/man1/lsof.md

```diff
@@ -1,26 +1,42 @@
 ## Name
 
-lsof
+lsof - List open files
 
 ## Synopsis
 
-```sh
-$ lsof [-p pid] [-d fd] [-u login/UID] [-g PGID] [filename]
+```**sh
+$ lsof [options]... [filename]
 ```
 
 ## Description
 
-List open files of a processes. This can mean actual files in the file system, sockets, pipes, etc.
+`lsof` lists opened files by certain processes, such as pipes, sockets, regular files...
```

Comment:
First I would say lsof lists all opened files. Then I would say there is the possibility to add arguments to filter these files by certain processes.